Makefile

```diff
@@ -10,15 +10,19 @@
 # only.
 #
 # By setting the variable FLOWDB_SERVICE you can choose which flowdb
-# version you'd like to use when running `make up`. Examples:
+# version or versions you'd like to use when running `make up`. Examples:
 #
-#     FLOWDB=flowdb-testdata make up
-#     FLOWDB=flowdb-synthetic-data make up
+#     FLOWDB_SERVICE=flowdb_testdata make up
+#     FLOWDB_SERVICE=flowdb_synthetic_data make up
+#     FLOWDB_SERVICE="flowdb_testdata flowdb_synthetic_data" make up
 #
+# flowmachine and flowapi will connected to the first flowdb service in the list.
 
 DOCKER_COMPOSE_FILE_DEV ?= docker-compose-dev.yml
-FLOWDB_SERVICES ?= flowdb-testdata
+export FLOWDB_SERVICES ?= flowdb_testdata
 DOCKER_SERVICES ?= flowapi flowmachine redis $(FLOWDB_SERVICES)
+export DB_HOST=$(word 1, $(FLOWDB_SERVICES))
```

Comment:
Ok, I see why the `export` is needed here - so that it is available as an environment env inside the flowmachine container.